{"uid":"635b46434e42d9a3","name":"Login Test with Incorrect Credentials","fullName":"test.test_3_login_user_incorrect_credentials#test_login_with_incorrect_credentials","historyId":"adac833431773c81fe306e827f919e91","time":{"start":1746788476470,"stop":1746788476470,"duration":0},"description":"Test for logging in with incorrect credentials and verifying error messages","descriptionHtml":"<p>Test for logging in with incorrect credentials and verifying error messages</p>\n","status":"broken","statusMessage":"ValueError: no option named 'playwright-browser'","statusTrace":"self = <_pytest.config.Config object at 0x7feb7a867d90>, name = 'playwright-browser', default = <NOTSET>, skip = False\n\n    def getoption(self, name: str, default=notset, skip: bool = False):\n        \"\"\"Return command line option value.\n    \n        :param name: Name of the option.  You may also specify\n            the literal ``--OPT`` option instead of the \"dest\" option name.\n        :param default: Default value if no option of that name exists.\n        :param skip: If True, raise pytest.skip if option does not exists\n            or has a None value.\n        \"\"\"\n        name = self._opt2dest.get(name, name)\n        try:\n>           val = getattr(self.option, name)\nE           AttributeError: 'Namespace' object has no attribute 'playwright-browser'. Did you mean: 'playwright_browser'?\n\n/home/user/.local/lib/python3.10/site-packages/_pytest/config/__init__.py:1693: AttributeError\n\nThe above exception was the direct cause of the following exception:\n\npytestconfig = <_pytest.config.Config object at 0x7feb7a867d90>\n\n    @pytest.fixture(scope=\"function\")\n    def browser(pytestconfig):\n        \"\"\"Запускаємо обраний браузер на основі CLI аргументу\"\"\"\n>       browser_name = pytestconfig.getoption(\"playwright-browser\")\nE       ValueError: no option named 'playwright-browser'\n\nconftest.py:18: ValueError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1746788476220,"stop":1746788476220,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1746788476220,"stop":1746788476220,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1746788476221,"stop":1746788476221,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1746788476221,"stop":1746788476221,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_session_faker","time":{"start":1746788476222,"stop":1746788476226,"duration":4},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"browser","time":{"start":1746788476471,"stop":1746788476471,"duration":0},"status":"broken","statusMessage":"ValueError: no option named 'playwright-browser'\n","statusTrace":"  File \"/home/user/.local/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/user/.local/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/user/.local/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/user/.local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/user/.local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/user/Documents/QA_Automation_projects/Pytest/Playwright - Pytest-Automationexercise/conftest.py\", line 18, in browser\n    browser_name = pytestconfig.getoption(\"playwright-browser\")\n  File \"/home/user/.local/lib/python3.10/site-packages/_pytest/config/__init__.py\", line 1704, in getoption\n    raise ValueError(f\"no option named {name!r}\") from e\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_3_login_user_incorrect_credentials"},{"name":"host","value":"user-VivoBook-15-ASUS-Laptop-X540UBR"},{"name":"thread","value":"96436-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_3_login_user_incorrect_credentials"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"635b46434e42d9a3.json","parameterValues":[]}